docs/user/openstack/customization.md

```diff
@@ -117,6 +119,66 @@ platform:
       clusterOSImage: my-rhcos
 ```
 
+## Additional Networks
+
+You can set additional networks for your machines by defining `additionalNetworks` parameter in the machine configuration. The parameter is a list of objects that contain a network `uuid`:
+
+```yaml
+additionalNetworks:
+- uuid: <network1_uuid>
+- uuid: <network2_uuid>
+```
+
+You can attach this parameter for both `controlPlane` and `compute` machines:
+
+Example:
+
+```yaml
+compute:
+- name: worker
+  platform:
+    openstack:
+      additionalNetworks:
+      - uuid: fa806b2f-ac49-4bce-b9db-124bc64209bf
+controlPlane:
+  name: master
+  platform:
+    openstack:
+      additionalNetworks:
+      - uuid: fa806b2f-ac49-4bce-b9db-124bc64209bf
+```
+
+**NOTE:** No allowed address pairs will be created for the additional networks.
+
+## Additional Security Groups
+
+You can set additional security groups for your machines by defining `additionalSecurityGroups` parameter in the machine configuration. The parameter is a list of strings with additional security group IDs:
+
+```yaml
+additionalSecurityGroups:
```

Comment:
Should be renamed `additionalSecurityGroupsIDs` for clarity.

Reply:
done